nav-bar.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>navigation bar</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"
+</head>
+<body>
+	<nav>
+		<a href="cv.html">HOME</a> |  <a href="education.html">Education</a> | <a href="booksIRead.html"> Books</a> | <a href="fav-site.html">Favorite Websites</a> | <a href="contact-me.html"> Enquiry</a>

```

Comment:
This is a case where you want to make use of `<ul>` and `<li>` within the navigation. 
Don't use the pipe separator ( `|`) ... that's a case of using markup or content for something that's presentational. 
What you want  is something more like:
```
<ul>
 	<li><a href="cv.html">HOME</a></li>
        <li><a href="education.html">Education</a></li>
         <li><a href="booksIRead.html"> Books</a></li>
         <li> <a href="fav-site.html">Favorite Websites</a></li>
          <li><a href="contact-me.html"> Enquiry</a></li>
</ul>